src/pivot-table/hooks/use-column-width.ts

```diff
@@ -287,18 +288,17 @@ export default function useColumnWidth(
 
   // The full scrollable width of the chart
   const totalWidth = useMemo(
-    () => leftGridWidth + rightGridFullWidth + GRID_BORDER,
-    [leftGridWidth, rightGridFullWidth],
+    () => leftGridWidthInfo.leftGridWidth + rightGridFullWidth + GRID_BORDER,
+    [leftGridWidthInfo.leftGridWidth, rightGridFullWidth],
   );
 
   const showLastRightBorder = useMemo(() => totalWidth < rect.width, [totalWidth, rect.width]);
 
   return {
-    leftGridWidth,
+    leftGridWidthInfo,
```

Comment:
I think I'd prefer same return structure as before. So we asign `leftGridWidth` and `getLeftGridColumnWIdth` here, and don't have to do the destructuring in PivotTable

Reply:
i can spread the result of leftGridWidthInfo in here and the hooks interface will extends that one like: 
```
interface ColumnWidthHook extends LeftGridWidthInfo { ... }
```

this will get rid of destructuring in pivot table!

Reply:
sounds good